docs/dynamic-plugins/faq.md

```diff
@@ -0,0 +1,154 @@
+# Frequently Asked Questions
+
+## What is Dynamic plugin derived package?
+
+This is a special JavaScript package that is derived from an original plugin package source code.
+You find more information about process of creating derived packages in the [Export Derived Dynamic Plugin Package](export-derived-package.md) document.
+The dynamic plugin derived packages shouldn't be pushed into the [public npm registry](https://www.npmjs.com), but it can be published to a private or internal npm registry.
+You can find more information about publishing dynamic plugins in the [Packaging Dynamic Plugins](packaging-dynamic-plugins.md) document.
+
+## How to configure up a dynamic plugin project to publish to a npm registry?
+
+See [Creating JavaScript package](packaging-dynamic-plugins.md#creating-javascript-package) in [Packaging Dynamic Plugins](packaging-dynamic-plugins.md) document.
+
+## How to configure RHDH to use private or internal npm registry?
+
+See [Using a custom NPM registry](installing-plugins.md#using-a-custom-npm-registry) in [Installing Plugins](installing-plugins.md) document.
+
+## How to install community Backstage plugin to the RHDH?
+
+1. Clone the community plugins repository.
+2. [Export](export-derived-package.md) the selected plugin as a derived dynamic plugin package.
+3. [Package and publish](packaging-dynamic-plugins.md) the derived dynamic plugin package.
+4. [Install](installing-plugins.md) the plugin in the RHDH. If plugin is frontend plugin you need to properly [wire it (define mount points and bindings)](frontend-plugin-wiring.md).
```

Comment:
maybe instead of "wire it (define mount points and bindings)" try "wire it by [defining mount points, dynamic routes, etc](link)." to remove the parenthesis?